package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azurepipelines/azdevops-kube-summary",
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Comment:
make this 3.5.2 and also lock file has to be udpate, when you do npm install lock file gets updated